modules/virt-virtctl-commands.adoc

```diff
@@ -29,70 +29,143 @@ To view a list of global command options that you can use with any `virtctl` com
+|`virtctl unpause vm\|vmi <vm_name>`
+|Unpause a VM or VMI.
 
 |`virtctl migrate <vm_name>`
-|Migrate a virtual machine.
+|Migrate a VM.
 
 |`virtctl restart <vm_name>`
-|Restart a virtual machine.
+|Restart a VM.
+|===
+
+[id='connecting-to-vms-and-vmis_{context}']
+== Connecting to VMs and VMIs
+
+You can use `virtctl` connect to the serial console, expose a port, set a proxy connection, specify a port, and open a VNC connection to a VM.
+
+.`virtctl console, expose, vnc` commands
+[width="100%",cols="1a,2a",options="header"]
+|===
+|Command |Description
+
+|`virtctl console <vmi_name>`
+|Connect to a serial console of a VMI.
 
 |`virtctl expose <vm_name>`
 |Create a service that forwards a designated port
-of a virtual machine or virtual machine instance and expose the service on
+of a VM or VMI and expose the service on
 the specified port of the node.
 
-|`virtctl console <vmi_name>`
-|Connect to a serial console of a virtual machine instance.
-
 |`virtctl vnc --kubeconfig=$KUBECONFIG <vmi_name>`
-|Open a VNC (Virtual Network Client) connection to a virtual machine instance. Access the graphical console of a virtual machine instance through a VNC which requires a remote viewer on your local machine.
+|Open a VNC (Virtual Network Client) connection to a VMI.
+
+Accessing the graphical console of a VMI through VNC requires a remote viewer on your local machine.
+
+|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only=true <vmi_name>`
+|Display the port number and connect manually to a VMI by using any viewer through the VNC connection.
 
-|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only=true <vmi-name>`
-|Display the port number and connect manually to the virtual machine instance by using any viewer through the VNC connection.
+|`virtctl vnc --kubeconfig=$KUBECONFIG --port=<port-number> <vmi_name>`
+|Specify a port number to run the proxy on the specified port, if that port is available.
+
+If a port number is not specified, the proxy runs on a random port.
+|===
 
-|`virtctl vnc --kubeconfig=$KUBECONFIG --port=<port-number> <vmi-name>`
-|Specify a port number to run the proxy on the specified port, if that port is available. If a port number is not specified, the proxy runs on a random port.
+[id='exporting-vms_{context}']
+== Exporting VM volumes
+
+You can use `virtctl vmexport` commands to create, download, or delete a volume exported from a VM, VM snapshot, or persistent volume claim (PVC).
+
+.`virtctl vmexport` commands
+[width="100%",cols="1a,2a",options="header"]
+|===
+|Command |Description
+|`virtctl vmexport create <vmexport_name> --vm\|snapshot\|pvc=<object_name>`
+|Create a `VirtualMachineExport` custom resource (CR) to export a volume from a VM, VM snapshot, or PVC.
 
+* `--vm`: Exports the PVCs and memory dump of a VM.
+* `--snapshot`: Exports the PVCs contained in a `VirtualMachineSnapshot` CR.
+* `--pvc`: Exports a PVC.
+* Optional: `--ttl=1h` specifies the time to live. The default duration is 2 hours.
```

Comment:
Optional: `--ttl=1h` specifies the time to live. The default duration is 2 hours. -> Optional: `--ttl=1h` specifies the duration, with a default value of 2 hours.